src/gui/plugins/view_angle/ViewAngle.qml

```diff
@@ -182,5 +183,130 @@ ToolBar {
         ViewAngle.OnAngleMode(0, 0, 1)
       }
     }
+
+    // set camera pose
```

Comment:
How about making this look a bit more like the pose fields in the component inspector? I think we should start being a bit more consistent with the UI across various widgets.

![campose](https://user-images.githubusercontent.com/5751272/122323449-f34cfa00-cedb-11eb-8181-ac851f62337d.gif)

That one also has the benefit of expanding, which is useful in case the numbers are very big.

Reply:
How's this? 0cdb38b3fdf9d3bebd4c2cc4a3d362103453de07

![Peek 2021-06-17 13-56](https://user-images.githubusercontent.com/8602001/122471750-7ae64780-cf74-11eb-8cf9-15840e681091.gif)

Reply:
Nice!